app/testOnly/connectors/CustomAuthConnector.scala

```diff
@@ -56,8 +56,14 @@ class CustomAuthConnector @Inject()(servicesConfig: ServicesConfig,
   override val serviceUrl: String = servicesConfig.baseUrl("auth-login")
   override def httpClientV2: HttpClientV2 = http
 
-  def login(nino: Nino, isAgent: Boolean, isSupporting: Boolean)(implicit hc: HeaderCarrier): Future[(AuthExchange, GovernmentGatewayToken)] = {
-    createPayload(nino, isAgent, isSupporting) flatMap {
+  def login(nino: String, isAgent: Boolean, isSupporting: Boolean)(implicit hc: HeaderCarrier): Future[(AuthExchange, GovernmentGatewayToken)] = {
```

Comment:
ideally if we can use Option[String] to represent nino parameter, but really really your design choice;